online-shopping-website/frontend/src/Components/Products/ProductDetails.js

```diff
@@ -75,8 +81,9 @@ const ProductButtons = (props) => {
                 </Button>
             </Stack>
             <h5 className="ProductDetails-ProductLimitText">Limit of 10 items per product in cart.</h5>
+            {/* {cookies.user && <p>{cookies.user}</p>} */}
             <Button className="ProductDetails-CartButton GreenButtonContained" variant="contained"
-                    endIcon={<AddShoppingCartIcon/>} onClick={AddToCart}>
+                    endIcon={<AddShoppingCartIcon/>}onClick={AddToCart} >
```

Comment:
Add a space between the `}` and the `onClick` just to make sure it works consistently